docusaurus.config.js

```diff
@@ -124,11 +124,12 @@ const config = {
         indexName: "first_index",
         contextualSearch: true,
         searchParameters: {},
+        searchPagePath: false,
       },
 
       metadata: [
         { name: "keywords", content: "Algomo, Algomo documentation" },
-        { name: "description", content: "Learning resources to help you get started on Algomo. Here you will learn how to use Ecommerce chatbot and improve user experience with AI chatbot."},
+        { name: "description", content: "Learning resources to help you get started on Algomo. Here you will learn how to use Algomo chatbot and improve user experience with it help."},

```

Comment:
I think this would be better:
```
Learning resources to help you get started on Algomo. Here you will learn how to use the Algomo chatbot and improve your users' experience.
```

Reply:
Okay, I've changed the description